components/net/fetch/methods.rs

```diff
@@ -317,7 +320,32 @@ fn basic_fetch(request: Rc<Request>, cache: &mut CORSCache) -> Response {
             }
         },
 
-        "blob" | "file" | "ftp" => {
+        "file" => {
+            if *request.method.borrow() == Method::Get {
+                match url.to_file_path() {
+                    Ok(file_path) => {
+                        let file_path_dup = &file_path.to_owned();
+                        if let Some(mut file) = File::open(file_path).ok() {
+                            let mut bytes = vec![];
+                            let _ = file.read_to_end(&mut bytes);
+                            let mime = guess_mime_type(file_path_dup);
+
+                            let mut response = Response::new();
+                            *response.body.lock().unwrap() = ResponseBody::Done(bytes);
+                            response.headers.set(ContentType(mime));
+                            response
+                        } else {
+                            Response::network_error()
+                        }
+                    },
+                    _ => Response::network_error()
+                }
+            } else {
+                Response::network_error()
```

Comment:
@KiChjang I have used `match` now but there are too many `Response::network_error()`. Do you prefer this?? I feel that it's bit worse.